docs/Systems/Tailwind/WorkingWithTailwind.stories.mdx

```diff
@@ -30,13 +36,34 @@ When building a page, ideally [Kaizen](https://github.com/cultureamp/kaizen-desi
 - `flex`: Adds `display: flex`
 - `justify-center`: Adds `justify-content: center`
 - `border-solid`: Adds `border-style: solid`
-- `p-16`: Adds `padding: 1rem` (`16px` = `1rem` - see the for more on this)
+- `p-16`: Adds `padding: 1rem` (`16px` = `1rem` - see the [layout and spacing docs](/story/systems-layout-and-spacing--page) for more on this)
 
 <br/>
 
 ## Altering Kaizen components with classnameOverride
 
-If a Kaizen component supports classNameOverride, Tailwind classes can also be used.
+If a Kaizen component supports classNameOverride, Tailwind classes can also be used.  
+This is particularly useful for adding margins to Kaizen components. In this example, our preset is being used to add `margin-right: 1.5rem` to Kaizen's `<Card/>` component, without the need to wrap it in a div.
+
+<Canvas>
+<div className="flex justify-center">
+  <Card classNameOverride="mr-24"> {/* Adds margin-right: 24px */} 
```

Comment:
IMO this isn't an ideal use case, to add a gap between items in flex they should use `.gap`. A bit more of a simplified example I think would be more effective, something like your CardContent where someone would want to add spacing between a header and paragraph:
```
<Heading variant="heading-3" classNameOverride="mb-24">{title}</Heading>
<Paragraph variant="body"> Lorem ipsum oogum boogum. Tutant meenage neetle teetles.</Paragraph>
```

Reply:
Yeah I see.

It looks like passing a `margin` rule to `classNameOverride` doesn't work for `<Heading/>`, it must be set already.

<img width="232" alt="image" src="https://user-images.githubusercontent.com/34709943/217128588-4c62cfc0-bb9f-4948-a553-886d740e4e83.png">

We'll have to come up with a different example